src/degree9/hooks.cljs

```diff
@@ -1,5 +1,8 @@
 (ns degree9.hooks
-  (:require [goog.object :as obj]))
+  (:require [goog.object :as obj]
+            [degree9.debug :as dbg]))
+
+(def ^:private debug (dbg "degree9:enterprise:server"))
```

Comment:
wrong debug statement

Reply:
Change statement to "hooks" instead of server.